### ISSUE TITLE
Frameworks adaptations

### ISSUE THREAD

**Issue by m5kr1pka** [m5kr1pka/canvas-scroll-clip#3]
Update documentation with implementations for a plugin in popular frameworks:

- VueJs (maybe Nuxt)
- React (maybe Gatsby, Next)
- Angular

**Comment by Vaqxin** [comment]
I agree. Trying to use this in next.js and Im getting some rendering issues. Last frame is still visible
![Screen Shot 2022-01-31 at 2 21 36 PM](https://user-images.githubusercontent.com/7260220/151858543-17353646-5bb1-409e-8407-e0f62b592641.png)

**Comment by m5kr1pka** [comment]
@Vaqxin Can you share the or part of the code? Try setting height and width of the container..

**Comment by MateuszJurus** [comment]
@Vaqxin Had the same issue, created a pull request with fix for that [here](https://github.com/m5kr1pka/canvas-scroll-clip/pull/10)

**Comment by m5kr1pka** [comment]
@MateuszJurus 👍

**Comment by JamesBrightman** [comment]
I second this suggestion - I'd like to use this in a React project but currently not having much luck implementing it myself. A clear copy-pasteable React example would be perfect.
